src/main/java/org/kohsuke/stapler/TaglibDocMojo.java

```diff
@@ -254,8 +254,7 @@ private void parseTagFile(File tagFile, Tag tag) throws IOException {
                 }
             }
         } catch (DocumentException e) {
-            IOException x = new IOException("Failed to parse " + tagFile);
-            x.initCause(e);
+            IOException x = new IOException("Failed to parse " + tagFile, e);
```

Comment:
NIT: could have inlined `x` now no need anymore for a variable to call `initCause()` on.